src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsuserpreferences/config/ResourceServerConfiguration.kt

```diff
@@ -1,21 +1,17 @@
 package uk.gov.justice.digital.hmpps.hmppsuserpreferences.config
 
-import com.nimbusds.jwt.JWTParser
-import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Profile
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
 import org.springframework.security.config.http.SessionCreationPolicy
-import org.springframework.security.oauth2.jwt.Jwt
-import org.springframework.security.oauth2.jwt.JwtDecoder
-import org.springframework.security.oauth2.jwt.MappedJwtClaimSetConverter
 import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter
 import org.springframework.security.oauth2.server.resource.authentication.JwtGrantedAuthoritiesConverter
 
 @Configuration
 @EnableWebSecurity
+@Profile("!unsecured")
```

Comment:
So this means that you have to specifically select to run a "unsecured" profile with PACT to avoid the auth issues. Ordinarily, you'd get these beans. Nice. Good work